src/docs/topics/publishing-releases/README.md

```diff
@@ -84,9 +84,4 @@ Let the whole world know about our shiny new release. Savor this part! These ste
 - [ ] Ask to publish a blog post on [.NET Foundation News](https://dotnetfoundation.org/news).
 - [ ] Tweet
 
-### After the release is done
```

Comment:
Why was it removed?

Reply:
both steps already exists in the same document is earlier steps